GiEnJul/Controllers/AdminController.cs

```diff
@@ -240,6 +240,25 @@ public async Task<ActionResult> DeleteRecipientAsync([FromBody] DeleteRecipientD
             return Ok();
         }
 
+        [HttpPut("recipient")]
+        [Authorize(Policy = "UpdateRecipient")]
+        public async Task<ActionResult> PutRecipientAsync([FromBody] PutRecipientDto recipientDto)
+        {
+            var recipientToPut = _mapper.Map<Recipient>(recipientDto);
+            var recipientToSet = await _recipientRepository.GetRecipientAsync(recipientDto.PartitionKey, recipientDto.RowKey);
+            foreach (var prop in recipientToPut.GetType().GetProperties())
+            {
+                var propValuePut = prop.GetValue(recipientToPut);
```

Comment:
propValuePut cease to exits after } , which means it could contain a simpler/generic name, i suggest `value` or `newValue`